example/eblob_backend.cpp

```diff
@@ -430,23 +474,19 @@ int blob_write_new(eblob_backend_config *c, void *state, dnet_cmd *cmd, void *da
 		}
 	} else {
 		[&] () {
-			if (eblob_read_return(b, &key, EBLOB_READ_NOCSUM, &wc))
-				return;
-
-			record_exists = true;
+			if (!(request.ioflags & DNET_IO_FLAGS_CAS_TIMESTAMP))
+				read_ext_headers(disk_ehdr, disk_jhdr);
```

Comment:
Let's make `read_ext_headers`once before [this if] (https://github.com/interiorem/elliptics/pull/35/files#diff-ae5c8480598d7c493d7314aadb6ae7d5R415) and only if `!(request.ioflags & DNET_IO_FLAGS_PREPARE) || (request.ioflags & DNET_IO_FLAGS_CAS_TIMESTAMP)`.